app/controllers/profiles_controller.rb

```diff
@@ -18,4 +15,10 @@ def show
     authorize profile
     render json: ProfileSerializer.new(profile)
   end
+
+  private
+
+  def resource
+    Profile
```

Comment:
I'm a bit confused. Are you using this method? And why not put these changes in the "IPP 12 - Change per-page/page to limit/offset and add links" commit where you add the Metadata and Pagination modules?